pkg/kubectl/cmd/set/set_volume_test.go

```diff
@@ -0,0 +1,282 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package set
+
+import (
+	"errors"
+	"testing"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
this should be removed, use versioned api

Reply:
use "k8s.io/client-go/kubernetes/typed/core/v1" replacing  "k8s.io/kubernetes/pkg/api"?

Reply:
@zjj2wry since I am still not clear with this version stuff

Reply:
api rep: `k8s.io/api...`
api maybe in: `k8s.io/api/core/v1`

Reply:
@zjj2wry  Is it a new move still on pushing? I notice that file in same package like set_env.go, set_image.go still use "k8s.io/kubernetes/pkg/api". 
since connections between them exist, if use core/v1 in set_volume.go. then it is necessary that  we should use the same in this whole package

Reply:
yes, will fix all set command :), and it is in process

Reply:
@zjj2wry so would mind if I just ignore this right now and fix it after other set command?
by the way , are there any PR that focus on this? please tell me so I can keep in track

Reply:
it's being actively worked on in https://github.com/kubernetes/kubernetes/pull/54554

@WanLinghao I would prefer not adding this until that PR is in

Reply:
@liggitt  OK, I will wait for this and rebase then